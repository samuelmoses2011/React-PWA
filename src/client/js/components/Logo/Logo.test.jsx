import React from 'react'
import { render } from 'enzyme'

import {Logo} from './Logo'

it('renders Logo correctly', () => {
  const test = render(<Logo />)
  expect(test).toMatchSnapshot()
})
