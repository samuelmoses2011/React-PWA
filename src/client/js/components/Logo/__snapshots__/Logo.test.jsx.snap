// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Logo correctly 1`] = `
<div
  class="logo_f4kazk7"
>
  HN
</div>
`;
